source/api_reference/index.html.md.erb

```diff
@@ -79,22 +77,22 @@ A successful response includes ``status`` and ``finished`` values:
   </tr>
   <tr>
     <td> submitted </td>
-    <td> Your user has submitted payment details but has not yet selected <b>Confirm</b>. </td>
+    <td> Your user has submitted payment details, and gone through 3D Secure.<br>The PSP has authorised the payment, but the user has not yet selected <b>Confirm</b>. </td>
```

Comment:
do we need to say 'if applicable' or something for 3dS?

Reply:
How about "Strong Customer Authentication" instead as that's the term we're using now?

Reply:
we should use that, yes, good point. but also it only applies to online payments not phone payments, so not all transactions with have SCA.